content/build/hydrobuild/faq.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hydrobuild FAQ
```

Comment:
Would be good to link to this page from the Hydrobuild section. Perhaps on the overview page?